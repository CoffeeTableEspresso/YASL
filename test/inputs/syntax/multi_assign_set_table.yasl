
const x = { .a: 1, .b: 2, .c: 3 }
let y = false

let z, y, x.a = .zzzz, .yy, 'xxx'

echo z
echo y
echo x


x.a, y, z = 'first', 'second', 'third'

echo z
echo y
echo x
