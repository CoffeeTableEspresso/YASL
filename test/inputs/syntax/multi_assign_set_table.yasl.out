zzzz
yy
{a: xxx, b: 2, c: 3}
third
second
{a: first, b: 2, c: 3}
